spotbugs/src/main/java/edu/umd/cs/findbugs/PluginLoader.java

```diff
@@ -851,7 +851,7 @@ private void loadPluginComponents()
                 String reports = detectorNode.valueOf("@reports");
                 String requireJRE = detectorNode.valueOf("@requirejre");
                 String hidden = detectorNode.valueOf("@hidden");
-                if (speed == null || speed.isEmpty()) {
+                if (speed == null || speed.length() == 0) {
```

Comment:
Why replace `isEmpty`? It seems like the better choice, O(1) instead of O(n).

Reply:
Oops, this got back when merging. Thanks for highlighting it!